src/commands/deploy.ts

```diff
@@ -104,6 +104,10 @@ export default class Deploy extends Command {
         } else if (!deployResult.success) {
             throw new Error(`Deployment error: ${ deployResult.error }`);
         }
+
+        if (deployResult.compilerErrors && deployResult.compilerErrors.length > 0) {
```

Comment:
Will this actually get to? Since a few lines above it checks whether the status of the response is unsuccessful.

Reply:
@graywolf336 the response is successful in some cases, this is one of those, when we have compiler errors, that's the problem this PR is solving.

Reply:
Ah makes sense :+1: